packs/nest/actions/humidity.py

```diff
@@ -0,0 +1,14 @@
+from lib import actions
+
+class HumidityAction(actions.BaseAction):
+    def run(self, structure, device, target=None):
+        if target:
+            target_humidity = target
+        else:
+            target_humidity = self._get_device(structure, device).humidity
```

Comment:
WFM. I think the truism we want to call out here is:

Be Explicit

This seems to be a common theme in our conversations internally and externally.